### ISSUE TITLE
[DOC] Update README docker build args

### ISSUE THREAD

**Issue by brhodes10** [rapidsai/clx#134]
## Report incorrect documentation

**Location of incorrect documentation**
README.md

**Describe the problems or issues found in the documentation**
Documentation references old build arg `image` 
```
docker build --build-arg image=rapidsai/rapidsai-dev-nightly:0.12-cuda9.2-devel-ubuntu18.04-py3.7 -t clx:latest .
```

**Steps taken to verify documentation is incorrect**
Reviewed Dockerfile

**Suggested fix for documentation**
Update README.md instructions to reflect new build args

```
ARG RAPIDS_VERSION=0.13
ARG CUDA_VERSION=10.1
ARG LINUX_VERSION=ubuntu18.04
ARG PYTHON_VERSION=3.7
```

**Comment by efajardo-nv** [comment]
Being fixed in PR #151.

**Comment by efajardo-nv** [comment]
Fixed by PR #151.
